plugins/GSdx/GSRendererCS.cpp

```diff
@@ -532,7 +532,7 @@ void GSRendererCS::Draw()
 				{"COLOR", 1, DXGI_FORMAT_R8G8B8A8_UNORM, 0, 28, D3D11_INPUT_PER_VERTEX_DATA, 0},
 			};
 
-			vector<unsigned char> shader;
+			vector<char> shader;
 			theApp.LoadResource(IDR_CS_FX, shader);
 			dev->CompileShader((const char *)shader.data(), shader.size(), "cs.fx", nullptr, "vs_main", macro, &vs.vs, layout, countof(layout), &vs.il);
```

Comment:
Casts could be removed since the vector doesn't use unsigned char anymore (though I could always do it (and std:: prefixing) later since it's in a Windows file).